src/SFA.DAS.ApplyService.Web/Controllers/Roatp/RoatpOverallOutcomeController.cs

```diff
@@ -64,6 +63,9 @@ public async Task<IActionResult> ProcessApplicationStatus(Guid applicationId)
                 case ApplicationStatus.Withdrawn:
                     return View("~/Views/Roatp/ApplicationWithdrawn.cshtml", model);
                 case ApplicationStatus.Removed:
+                    var oversightStatus = await _overallOutcomeService.GetOversightReviewStatus(applicationId);
```

Comment:
You've added a new method `GetOversightReviewStatus` and used it here, but why not just use `GetOversightReview` instead? That would save adding new methods, and the performance difference would be negligible.

Also it seems odd to get the review using the api client, but the status via the overall outcome service, no?

Reply:
yeah, I'm mixing and matching.  Will conform